src/test/java/org/wfanet/anysketch/SketchProtosTest.java

```diff
@@ -62,7 +60,9 @@ public long apply(String item, Map<String, Long> itemMetadata) {
               new ValueFunction(
                   "value-function-1", Aggregators.sum(), Distributions.oracle("feature1", 5, 105)),
               new ValueFunction(
-                  "value-function-2", Aggregators.unique(), Distributions.oracle("feature2", 6, 15))));
```

Comment:
nit, the style changes from the first valueFunction to the second (single to multi-line).

Reply:
That is auto formatted by a magic "git fix" command, I think it is also what lint wants

Reply:
To give further background, we use google-java-format to format all Java files. The GitHub lint workflow check will enforce this.

We should likely update a GitHub page with the formatter info. https://github.com/world-federation-of-advertisers/cross-media-measurement/blob/main/docs/code-style.md has some info, but that repo has no Java code.